### ISSUE TITLE
Feature request : Role and Permission for Labelling tool

### ISSUE THREAD

**Issue by kriti-kumari** [microsoft/OCR-Form-Tools#619]
**Is your feature request related to a problem? Please describe.**
We have different team for labelling data and different for maintaining critical information such as SAS key for blob storage. Project should be listed and labelling team should have only access to label and train data and should not have access to SAS keys as that is confidential information.

Considering the fact that labelling tool should have read, write , list and delete access for blob storage using SAS key, someone can use this SAS key for modifying blob storage using other tools like AZ copy, storage explorer etc.  

As recent project and  connection are stored in browser local storage and not in data base we need to share project every for other member of teams to use.

These issues could be resolved with roles and permissions.

**Describe the solution you'd like**
Different roles and permission in labelling tool . For instance:
	1. Admin : Have access to all the options.
	2. Labelling member : Should have only option to label and train data. Shouldn't have option to view SAS key or  security token

**Describe alternatives you've considered**
	1. Share project
	2. Creating SAS key for short duration . 

**Additional context**
This will resolve below issues :
	1. Independence of using any browser in a system
	2. Independence of using any system without have share project string.
	3. Multiple teams can work without critical information being compromised.
        4. Better data security.

Please do let me know if we have other solutions

**Comment by xinase** [comment]
thank you for the feedback. 
when a user could label and train data,  it implies that such user could have read/write permission to the azure blob storage, would that be a concern for you?

**Comment by kriti-kumari** [comment]
Yes, Actually we are trying to use AFR for a critical project were data is confidential. We have a team to train data , who should have access to read and label data , but not to download or upload or do any modification with blob storage. 

labelling tool should have read, write , list and delete access for blob storage using SAS key, but we don't want to give these access to labelling team.

Can we preconfigure SAS key and hide "Connections" option for labelling tool for few users . Also projects should be pre populated.

**Comment by kriti-kumari** [comment]
Hi team, any update on this

**Comment by kriti-kumari** [comment]
Hi  team, we are trying to use form recognizer for our project but due to this limitation in labeling tool we are unable to get approval from InfoSec (Security) team. It would be really helpful if there is any suggestion regarding this.

**Comment by fpousserot** [comment]
I vote for this request, because we have exactly the same no go from the security teams
